examples/glusterfs/README.md

```diff
@@ -67,13 +67,14 @@ You can verify that the pod is running:
 
 ```shell
 $ kubectl get pods
-POD         IP            CONTAINER(S)   IMAGE(S)              HOST                                  LABELS    STATUS    CREATED          MESSAGE
-glusterfs   10.244.2.13                                        kubernetes-minion-151f/23.236.54.97   <none>    Running   About a minute   
-                          glusterfs      kubernetes/pause                                                      Running   About a minute   
+NAME             READY     STATUS    RESTARTS   AGE
+glusterfs        1/1       Running   0          3m
 
+$ kubectl get pods glusterfs -t {{.status.hostIP}}
```

Comment:
When I try this command, linebreak (\n) is missing so it looks weird to me. 

`$ cluster/kubectl.sh get pods redis-master-lc771 -t {{.status.hostIP}}`
`10.240.163.41$ `